README.md

```diff
@@ -9,3 +9,152 @@
 ## UNDER CONSTRUCTION
 
 This repository contains the work-in-progress code for the upcoming revision of [gardenctl](https://github.com/gardener/gardenctl), [Gardener](https://gardener.cloud/)'s command-line client.
+
```

Comment:
remove under construction section

Reply:
ok, done. I kept it in because I wanted to ask you if you think that we should add a hint somewhere that this is the successor of gardenctl-v1? I think we will rather add a deprication message to the gardenctl-v1 repository once we go stable. WDYT?